src/item.rs

```diff
@@ -31,13 +31,38 @@ impl ItemBundle {
         }
     }
 
-    pub fn spawn(mut commands: EntityCommands, item_spawn_meta: &ItemSpawnMeta) {
+    pub fn spawn(
+        mut commands: EntityCommands,
+        item_spawn_meta: &ItemSpawnMeta,
+        items_assets: &mut ResMut<Assets<ItemMeta>>,
+    ) {
         let ground_offset = Vec3::new(0.0, consts::GROUND_Y, consts::ITEM_LAYER);
         let transform_bundle = TransformBundle::from_transform(Transform::from_translation(
             item_spawn_meta.location + ground_offset,
         ));
 
         commands.insert_bundle(transform_bundle);
+
+        if let Some(item_meta) = items_assets.get_mut(&item_spawn_meta.item_handle) {
+            if let ItemKind::BreakableBox {
+                hurtbox,
+                hits,
+                item,
+                ..
+            } = &item_meta.kind
+            {
+                let mut physics_bundle = PhysicsBundle::new(hurtbox, BodyLayers::ENEMY);
+                physics_bundle.collision_groups.filters = BodyLayers::PLAYER_ATTACK;
```

Comment:
we should probably add a new bodylayer in `impl BodyLayers` in collisions.rs  `pub const BREAKABLE_ITEM: u32 = 1 << 4;` and use that here as the membership, keeping the filter as `BodyLayers::PLAYER_ATTACK` as you have it.